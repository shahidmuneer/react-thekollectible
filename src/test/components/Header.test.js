import {render} from '../utils'
import Header from '../../components/header'
import React from 'react'

describe('<Header /> component', () => {
    it('renders with brand name "conduit"', () => {
        const wrapper = render(<Header />)
        expect(wrapper.getByText('conduit')).toBeDefined()
      })
    it('renders with 3 navbar item', () => {
      const wrapper = render(<Header />)
      const navItems= wrapper.container.getElementsByClassName('nav-item')
      expect(navItems.length).toEqual(3)
      expect(wrapper.asFragment()).toMatchSnapshot()
    })
})