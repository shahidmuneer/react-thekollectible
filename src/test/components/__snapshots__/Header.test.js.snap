// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header /> component renders with 3 navbar item 1`] = `
<DocumentFragment>
  <nav
    class="navbar navbar-light"
  >
    <div
      class="container"
    >
      <a
        class="navbar-brand"
        href="/"
      >
        conduit
      </a>
      <ul
        class="nav navbar-nav pull-xs-right"
      >
        <li
          class="nav-item"
        >
          <a
            class="nav-link active"
            href="/"
            style="text-transform: capitalize;"
          >
            Home
          </a>
        </li>
        <li
          class="nav-item"
        >
          <a
            class="nav-link "
            href="/login"
            style="text-transform: capitalize;"
          >
            Login
          </a>
        </li>
        <li
          class="nav-item"
        >
          <a
            class="nav-link "
            href="/sign-up"
            style="text-transform: capitalize;"
          >
            Sign Up
          </a>
        </li>
      </ul>
    </div>
  </nav>
</DocumentFragment>
`;
